src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String line){
+        this.description = line;
+    }
+
+    public void markAsDone(){
```

Comment:
Good job following the naming standards for all the methods (use of camelCase) :)